playbooks/roles/setup_pipeline/tasks/main.yml

```diff
@@ -131,3 +131,8 @@
   become_user: "{{ pipeline_user }}"
   tags:
     - install
+
+- name: Adding hadoop user to pipeline group
```

Comment:
@mrgnr No, we need this, because hadoop needs to create some dirs on this folder while the pipeline is running.